app/assets/javascripts/stores/notification_store.js

```diff
@@ -87,6 +87,17 @@ const NotificationStore = Flux.createStore(storeMethods, (payload) => {
       handleErrorNotification(data, payload.actionType);
       NotificationStore.emitChange();
       break;
+    case 'NEEDS_UPDATE':
+      addNotification({
+        message: payload.data.result,
+        closable: true,
+        type: 'success'
+      });
+
+      $('html, body').animate({scrollTop: 0}, 200)
+        .promise()
+        .then(() => NotificationStore.emitChange());
+      break;
```

Comment:
Although this makes sense on it's own in the current UI, this should be addressed by fixing the general problem of notifications appearing when the visible page doesn't include the Notifications section: #234. I think we'll work on a fix for that one soon, so let's cut out the animation here.